H2-migration/ImportCSV.sql

```diff
@@ -0,0 +1,279 @@
+#Load oc_series_elements
+LOAD DATA INFILE "/Users/orlo/Documents/Current_Projects/ELAN/Opencast/h2/oc_series_elements.csv"  
```

Comment:
The path is always hardcoded to your environment. Please change that to a relative path and add a readme on how to use this.